client/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Bootstrap -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="./assets/styles/styles.css">
+
+    <title>Futureproof Search</title>
+</head>
+
+<body>
+
+    <main>
+        <div class="container-fluid mt-1">
+            <div class="row justify-content-center">
+                <div class="col-7 col-sm-6 col-md-5 col-xl-4">
+                    <img id="logo" src="./assets/images/logo.svg" alt="Google logo">
+                </div>
+            </div>
+
+            <form>
+                <div class="row justify-content-center">
+                    <div class="col-9 col-sm-8 col-md-7 col-lg-6 col-xl-5">
+                        <div class="search-bar">
+                            <img id="search-logo" src="./assets/images/search-logo.svg" alt="Search bar logo">
+                            <input id="search-bar" type="text">
```

Comment:
For your search bar you can add the attribute 'autofocus' and when the page loads the user will be directed to typing in the search bar automatically.